.github/workflows/pkgci_test_android.yml

```diff
@@ -31,7 +31,7 @@ permissions:
   contents: read
 
 jobs:
-  cross_compile:
+  android_arm64:
```

Comment:
Just noticed that the Android and RISCV cross compile jobs are sharing a cache (named "cross_compile") when they shouldn't. Pushed another commit that gives each a unique cache key.